src/BIAnalyticsService/CreateDataAssets/CreateDataAssets.cs

```diff
@@ -0,0 +1,198 @@
+        if (string.IsNullOrEmpty(screeningEpisodeUrl) || string.IsNullOrEmpty(screeningProfileUrl))
+        {
+            _logger.LogError("One or both URLs are not configured. \nUrl:{screeningProfileUrl}\nUrl:{screeningEpisodeUrl}", participantUrl, screeningEpisodeUrl);
+            return req.CreateResponse(HttpStatusCode.InternalServerError);
+        }
+
+        try
+        {
+            await SendToCreateParticipantScreeningEpisodeAsync(episode, screeningEpisodeUrl);
+            await SendToCreateParticipantScreeningProfileAsync(participant, screeningProfileUrl);
+        }
+        catch (Exception ex)
+        {
+            _logger.LogError(ex, "Failed to create episode or profile.");
+            return req.CreateResponse(HttpStatusCode.InternalServerError);
+        }
+
+        return req.CreateResponse(HttpStatusCode.OK);
+    }
+
+    private (string screeningEpisodeUrl, string screeningProfileUrl) GetConfigurationUrls()
+    {
+        return (Environment.GetEnvironmentVariable("CreateParticipantScreeningEpisodeUrl"), Environment.GetEnvironmentVariable("CreateParticipantScreeningProfileUrl"));
+    }
+
+    private async Task SendToCreateParticipantScreeningProfileAsync(Participant participant, string screeningProfileUrl)
+    {
+        if (participant != null)
+        {
+            _logger.LogInformation("Mapping participant profile data.");
+
+                var screeningProfile = new ParticipantScreeningProfile
+                {
+                    NhsNumber = participant.nhs_number,
+                    ScreeningName = String.Empty,
+                    PrimaryCareProvider = String.Empty,
+                    PreferredLanguage = participant.preferred_language,
+                    ReasonForRemoval = participant.removal_reason,
+                    ReasonForRemovalDt = String.Empty,
+                    NextTestDueDate = participant.next_test_due_date,
+                    NextTestDueDateCalculationMethod = participant.ntdd_calculation_method,
+                    ParticipantScreeningStatus = participant.subject_status_code,
+                    ScreeningCeasedReason = String.Empty,
+                    IsHigherRisk = participant.is_higher_risk,
+                    IsHigherRiskActive = participant.is_higher_risk_active,
+                    HigherRiskNextTestDueDate = participant.higher_risk_next_test_due_date,
+                    HigherRiskReferralReasonCode = participant.higher_risk_referral_reason_code,
+                    HrReasonCodeDescription = String.Empty,
+                    DateIrradiated = participant.date_irradiated,
+                    GeneCode = participant.gene_code,
+                    GeneCodeDescription = String.Empty,
+                    RecordInsertDatetime = DateTime.Now.ToString()
+                };
+
+                string serializedParticipantScreeningProfile = JsonSerializer.Serialize(screeningProfile, new JsonSerializerOptions { WriteIndented = true });
```

Comment:
Let's not add indentation to the serialised json, it might make it more readable in the logs but it's going to increase the size of the request payload